google/auth/crypt.py

```diff
@@ -239,6 +264,7 @@ def from_service_account_file(cls, filename):
         Returns:
             Signer: The constructed signer.
         """
-        from google.auth import _service_account_info
-        _, signer = _service_account_info.from_filename(filename)
-        return signer
+        with io.open(filename, 'r', encoding='utf-8') as json_file:
```

Comment:
> Why do we care about UTF-8?

Because json files should be in utf-8 as per the spec, and we don't want the system default encoding.
 
> In Python 3 json.load eats bytes or unicode?

I don't know about that, I've always passed in a file in text mode.